src/app/public/modules/repeater/fixtures/repeater-missing-tag.fixture.ts

```diff
@@ -0,0 +1,27 @@
+
+import {
+  Component
+} from '@angular/core';
+
+/** Simple component for testing an orderable repeater with missing tags (error state). */
```

Comment:
Does putting this in a JSDoc comment mean that it could end up in the docs? If so, do we need to clean this text up? For example, this isn't a complete sentence. Also, we say "orderable" here, but our terminology elsewhere is consistently "reorderable."

Reply:
This is just a testing fixture so it won't show in the docs. FWIW, I've still changed the `orderable` to `reorderable` :)